public/css/styles.css

```diff
@@ -0,0 +1,673 @@
+/*---general page and components css---*/
+html {
+  box-sizing: border-box;
+}
+
+*, *:before, *:after {
+  box-sizing: border-box;
+}
+
+* {
+  overflow: auto;
+}
+
+body {
+  width: 100%;
+  margin: auto;
+  font-family: Helvetica, Arial, "Lucida Grande", sans-serif;
+  font-weight: 300;
+}
+
+nav {
+  margin-left: -1em;
+}
+nav li {
+  display: inline-block;
+}
+
+nav, section {
+  text-align: center;
+ }
+header > p {
+    margin-bottom: -.7em;
+  }
+section {
+  width: 100%;
+  padding-left: 5%;
+  padding-right: 5%;
+  padding-top: 2em;
+}
+
+ul {
+    margin-left: -1em;
+}
+
+img {
+  max-width: 100%;
+}
+
+h1 {
+  position: relative;
+  overflow: hidden;
+  z-index: 1;
+  margin-bottom: 2em;
+  margin-top: 2em;
+}
+
+/* -- lines created on h1 -- */
+
+h1:after, h1:before {
+  top: 51%;
+  width: 32%;
+  content: "";
+  position: absolute;
+  overflow: hidden;
+  height: 1px;
+  background-color: #585050;
+  margin-left: 3%;
+}
+
+h1:before {
+  margin-left: -35%;
+  text-align: right;
+}
+
+header {
+  margin-top: 3em;
+  margin-bottom: -3em;
```

Comment:
Again, negative margins are a code smell and are generally a warning sign that something is wrong. In this case, I think you're having to pull up on this element because of the poorly sized underline SVG. If that were sized correctly, you probably wouldn't need to make this adjustment.